### DISCUSSION TITLE
Missing onError handler for invocation 'building-schema', error was 'Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/

### DISCUSSION THREAD

**Discussion by andreisemianiuk** [gatsbyjs/gatsby#33892] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

I  have a problem then try to add gatsby-source-wordpress plugin to my project. It's also repeats in another starter projects by gatsby.

Missing onError handler for invocation 'building-schema', error was 'Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/ but
"WpTemplate_Контакты" does not.'. Stacktrace was 'GraphQLError: Names must match /^[_a-zA-Z][_a-zA-Z0-9]*$/ but
"WpTemplate_Контакты" does not.
    at isValidNameError
(E:\reactProjects\gatsby-projects\bug-repro\node_modules\graphql\utilities\assertValidName.js:42:12)
    at assertValidName
(E:\reactProjects\gatsby-projects\bug-repro\node_modules\graphql\utilities\assertValidName.js:21:15)
    at checkIsAllowedTypeName
(E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\schema.js:573:3)
    at forEach (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\schema.js:284:9)
    at Array.forEach (<anonymous>)
    at addTypes (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\schema.js:246:9)
    at updateSchemaComposer (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\schema.js:123:9)
    at buildSchema (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\schema.js:64:9)
    at build (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\schema\index.js:112:18)
    at buildSchema (E:\reactProjects\gatsby-projects\bug-repro\node_modules\gatsby\src\services\build-schema.ts:19:3)'

### Reproduction Link

https://github.com/andreisemianiuk/gatsby-bug-report

### Steps to Reproduce

1. Create  Gatsby project
2. Install gatsby-source-wordpress plugin
3. Add gatsby-source-wordpress plugin to gatsby-config.js 
{
      resolve: `gatsby-source-wordpress`,
      options: {
       url:
          process.env.WPGRAPHQL_URL ||
          `http://ecosiprom.com/graphql`,
      },
    }

### Expected Result

I want to start project with wordpress colloboration

### Actual Result

I always receive error

### Environment

```shell
System:
    OS: Windows 10 10.0.18363
    CPU: (4) x64 Intel(R) Core(TM) i3-6100U CPU @ 2.30GHz
  Binaries:
    Node: 14.17.3 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.11 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.1.2 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: 76.0.3809.132
    Edge: Spartan (44.18362.1593.0)
  npmPackages:
    gatsby: ^4.1.0 => 4.1.0
    gatsby-source-wordpress: ^6.1.0 => 6.1.0
  npmGlobalPackages:
    gatsby-cli: 4.0.0
```

**Comment by LekoArts** [comment]
Hi!

Your template name `WpTemplate_Контакты` doesn't satisfy the regex `/^[_a-zA-Z][_a-zA-Z0-9]$/`. You must not use `Контакты` but have to use the latin alphabet (not Cyrillic)

**Reply by andreisemianiuk** [reply]
But I can't find there I can fix it. In WP dashboard or in Gatsby, or in PHP code.
